Makefile

```diff
@@ -44,7 +44,6 @@ notice:
 .PHONY: check-ci
 check-ci:
 	@mage update
-	@mage check
```

Comment:
This shouldn't be here, this actually makes the linter run twice and it should only be run by the github action.